src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,95 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  const [activePage, setActivePage] = useState('1');
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: activePage === 'prev',
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? true : false}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+
+            setActivePage('prev');
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {Array(numberOfPages)
+        .fill(null)
+        .map((_, index) => {
+          return (
+            <li
+              key={index + 1}
+              className={cn('page-item', {
+                active: activePage === `${index + 1}`,
+              })}
+            >
+              <a
+                data-cy="pageLink"
+                className="page-link"
+                href="#1"
```

Comment:
The `href` attribute for pagination should not be a static '#1'. It's typically better to use 'javascript:void(0);' or a button element to prevent the default anchor behavior if you are handling the click event.